helm/alfresco-content-services/charts/alfresco-content-common/templates/_helpers-image-pull-secrets.tpl

```diff
@@ -0,0 +1,7 @@
+{{- define "alfresco-content-services.imagePullSecrets" }}
+{{- if .Values.global.alfrescoRegistryPullSecrets }}
+# only set this secret if a private docker registry variable is defined 1
```

Comment:
```suggestion
```

not sure what was the reason for that but don't seems very useful and worth keeping